src/main/java/org/usfirst/frc/team2473/framework/CHS_SparkMax.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc.team2473.framework;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class CHS_SparkMax {
```

Comment:
What is the purpose of this class and why would I use it over CANSparkMax?

Reply:
We need to return the last non-zero encoder value, so we created a wrapper for CANSparkMax to put our own code.